networkx/algorithms/bipartite/matching.py

```diff
@@ -415,7 +417,11 @@ def _connected_by_alternating_paths(G, matching, targets):
         for v in G
         if v in targets
         or _is_connected_by_alternating_path(
-            G, v, matched_edges, unmatched_edges, targets
+            G,
+            v,
+            matched_edges,
+            unmatched_edges,
+            targets,
```

Comment:
similarly here -- this should pass the linters as one line of arguments...

Reply:
Ditto ^